apps/consumer-client/src/constants.ts

```diff
@@ -52,3 +53,7 @@ export const CONSUMER_SERVER_URL = CONSUMER_SERVER_URL_ENV
   : IS_STAGING
   ? "https://consumer-server-staging.onrender.com/"
   : "http://localhost:3003/";
+
+export const GPC_ARTIFACT_CONFIG = parseGPCArtifactsConfig(
+  process.env.GPC_ARTIFACTS_CONFIG_OVERRIDE
```

Comment:
i think this is never `undefined`, but perhaps sometimes the string `'undefined'` (based on the above `build.ts`)

Reply:
fwiw this is how we do optional env vars in `passport-client` - https://github.com/proofcarryingdata/zupass/blob/artwyman/experimental-artifacts/apps/passport-client/build.ts#L24-L30, and idt there's a good reason not to do the same type of thing in `consumer-client`

Reply:
The string "undefined" is actually checked for in parseGPCArtifactsConfig.